spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -15,7 +15,7 @@ we go.
 
 You need Java installed (Java 7 or better, we recommend Java 8), and to build, you need to have Maven installed as well.
 
-You also need to have link:http://redis.io/[Redis] installed and running if you plan on running a local system, or to run the included tests.
+You also need to have link:http://redis.io/[Redis] and link:http://kafka.apache.org/[Apache Kafka] installed and running if you plan on running the Local server, or to run the included tests.
```

Comment:
don't need kafka running for the local server/tests, do need it if running app w/ kafka as the binder.